Assets/Scripts/QuitOnEscape.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+
+public class QuitOnEscape : MonoBehaviour
+{
+    void Update()
+    {
+        if (Input.GetKey ("escape"))
```

Comment:
Maybe use https://docs.unity3d.com/ScriptReference/Input.GetButton.html and add the key to the input list
Also, remove the space between GetKey and the parenthesis